tests/fs/test_strip_unstrip.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from dvc_objects.fs.implementations import (
+    azure,
+    gs,
+    hdfs,
+    http,
+    https,
+    oss,
+    s3,
+    ssh,
+    webhdfs,
+)
+from dvc_objects.fs.base import FileSystem
+
+
+@pytest.fixture(autouse=True)
+def mock_check_requires(mocker):
+    yield mocker.patch.object(FileSystem, "_check_requires")
+
+
+@pytest.mark.parametrize(
+    "fs_cls, urlpath, stripped",
+    [
+        (azure.AzureFileSystem, "azure://container", "container"),
+        (gs.GSFileSystem, "gs://container", "container"),
```

Comment:
Just wondering if we also need it for `gdrive` remotes

Reply:
My intention was to only cover object-based filesystems, and leave others out. As `_strip_protocol` and `unstrip_protocol`'s future is not quite clear. :)